src/components/atoms/Button/AddNoticeButton/AddNoticeButton.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Link } from 'components/atoms';
+import { useTranslation } from 'react-i18next';
```

Comment:
External import should be above our imports.

Reply:
oops, thx